hack/lib/init.sh

```diff
@@ -41,7 +41,11 @@ KUBE_RSYNC_COMPRESS="${KUBE_RSYNC_COMPRESS:-0}"
 
 # Set no_proxy for localhost if behind a proxy, otherwise,
 # the connections to localhost in scripts will time out
-export no_proxy=127.0.0.1,localhost
+if [ ! -n "$no_proxy" ]; then
```

Comment:
we can do this simpler like:
```bash
export no_proxy="127.0.0.1,localhost${no_proxy:+,${no_proxy}}"
```
no need for the if block.

Reply:
Thank you very much for your suggestion. 
Done.